docs/document/docs/guides/format-setting-plugin.md

```diff
@@ -97,6 +97,9 @@ Create a manifest file like the one below.
 Kintone UI Component will be used for this customization on the plug-in settings page, so describe the path of the UMD file in the config object.<br>
 See [Quick Start](../getting-started/quick-start.md) for how to upload a UMD file.
 
+## Create a Plug-in Icon
+- Create a plug-in icon following the [guide](https://kintone.dev/en/plugins/introduction-to-plug-ins/steps-for-plug-in-development/#creating-each-file).<br>
+- Place the icon file (icon.png) in the image folder.
```

Comment:
Please change section title to lower case.
Please write the contents without bullet points.

```suggestion
## Create a plug-in icon
Create a plug-in icon following the [guide](https://kintone.dev/en/plugins/introduction-to-plug-ins/steps-for-plug-in-development/#creating-each-file).<br>
Place the icon file (icon.png) in the image folder.
```